lumicks/pylake/scan.py

```diff
@@ -337,6 +339,7 @@ def plot_correlated(
             scan.plot_correlated(file.force1x, channel="red")
         """
         from lumicks.pylake.nb_widgets.correlated_plot import plot_correlated
+
```

Comment:
does `black` add this?

Reply:
That was isort, which unfortunately does not restrict itself to the module wide imports. Reverted it.